specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2020-09-01-preview/sqlmigration.json

```diff
@@ -0,0 +1,1718 @@
+          }
+        },
+        "x-ms-examples": {
+          "Get Database Migration resource.": {
+            "$ref": "./examples/SqlMiGetDatabaseMigration.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
>"put": [](http://example.com/codeflow?start=6&length=6)

No PATCH? If a partial update is supported, it must be via PATCH.

[https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc009-use-put-for-update-of-entire-resource-ie-replace-use-patch-only-for-partial-update-of-resource](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc009-use-put-for-update-of-entire-resource-ie-replace-use-patch-only-for-partial-update-of-resource)

Reply:
Currently we don't support partial update on Database Migration resource. @hitenjava can you please add further details on this.

Reply:
Correct - databaseMigrations resource is a proxy resource and we don't support partial update on databaseMigrations resource.